sources/engine/Stride.Graphics/GeometricPrimitives/GeometricPrimitive.Torus.cs

```diff
@@ -117,16 +117,18 @@ public static GeometricPrimitive New(GraphicsDevice device, float majorRadius =
             /// <exception cref="System.ArgumentOutOfRangeException">tessellation;tessellation parameter out of range</exception>
             public static GeometricMeshData<VertexPositionNormalTexture> New(float majorRadius = 0.5f, float minorRadius = 0.16666f, int tessellation = 32, float uScale = 1.0f, float vScale = 1.0f, bool toLeftHanded = false)
             {
-                var vertices = new List<VertexPositionNormalTexture>();
-                var indices = new List<int>();
-
-                if (tessellation < 3)
-                    tessellation = 3;
+                tessellation = Math.Max(3, tessellation);
 
                 int stride = tessellation + 1;
 
+                VertexPositionNormalTexture[] vertices = new VertexPositionNormalTexture[stride * stride];
+                int[] indices = new int[tessellation * tessellation * 6];
```

Comment:
In the original version, `var` was used on the left side of the declaration to avoid duplication, as the right side already defines the type.

Why have you opted here to specify the type in both left and right parts of the assignment?

Reply:
emmm,It doesn't matter.  the better importance is  original use List,but actually,the array size is definite ,we can optimize it

Reply:
Yeah, I know it doesn't matter. It's a stylistic choice.

Just asking because the original used `var` and (in my opinion) removes some redundancy and reads better.

Reply:
I modified it